pose_frame_semantics/tutorial.md

```diff
@@ -1,23 +1,82 @@
 # Pose Frame Semantics Tutorial
 
 This documentation explains the implemented changes proposed by
-[Pose Frame Semantics Proposal](http://sdformat.org/tutorials?tut=pose_frame_semantics_proposal) in SDFormat 1.7.
+[Pose Frame Semantics Proposal](http://sdformat.org/tutorials?tut=pose_frame_semantics_proposal) in SDFormat 1.7,
+using visuals from the [Pose Frame Specification for SDFormat 1.7](https://doi.org/10.36288/ROSCon2019-900911)
+presentation at [ROSCon 2019](https://roscon.ros.org/2019/).
 
 Prerequisites:
 
 - [Specifying Pose in SDFormat](http://sdformat.org/tutorials?tut=specify_pose&cat=specification&#specifying-pose-in-sdformat)
 
 - [Specifying model kinematics in SDFormat](http://sdformat.org/tutorials?tut=spec_model_kinematics&cat=specification&#specifying-model-kinematics-in-sdformat)
 
+## Syntax
+
+This document uses [XPath syntax](https://www.w3schools.com/xml/xpath_syntax.asp)
+to describe elements and attributes concisely.
+For example, `<model>` tags are referred to as `//model` using XPath.
+XPath is even more concise for referring to nested tags and attributes.
+In the following example, `<link>` elements inside `<model>` tags are
+referenced as `//model/link` and  model `name` attributes as `//model/@name`:
+
+    <model name="model_name">
+      <link/>
+    </model>
+
 ## What's New In SDFormat 1.7 With Examples
 
+The primary new features in SDFormat 1.7 are specifying poses relative to a
+frame of your choice using `//pose/@relative_to` and creating named frames
```

Comment:
maybe add the word `explicit` since adding `<frame>` tag is essentially explicitly defining frames? But if you purposely left out the imlicit/explicit stuff for later, I can see how you want to keep this part simple.

suggestions:
"and creating an explicit name frames using `//frame`"

Reply:
I'd rather not introduce that complexity yet. Let's save it for later in the document